package.json

```diff
@@ -39,18 +44,15 @@
     "@nextcloud/browserslist-config": "^3.0.0",
     "@nextcloud/eslint-config": "^8.2.0",
     "@nextcloud/l10n": "^1.4 || ^2.0",
-    "@nextcloud/vue": "^8.6.2",
-    "@vitejs/plugin-vue2": "^2.2.0",
-    "browserslist-to-esbuild": "^2.1.0",
+    "@nextcloud/vite-config": "^1.2.2",
     "sass": "^1.57.1",
     "vite": "^5.0.7",
-    "vite-plugin-css-injected-by-js": "^3.0.0",
     "vue": "^2.7.8",
     "vue-template-compiler": "^2.7.8"
   },
   "peerDependencies": {
     "@nextcloud/l10n": "^1.4 || ^2.0",
-    "@nextcloud/vue": "^8.0.1",
+    "@nextcloud/vue": "^8.6.2",
```

Comment:
Do we want to enforce this? It is basically a library so we are compatible with everything >= 8.0.1 and could support what ever the using app is providing.

Reply:
It was done already in `devDependencies`, I only make them in-line.

But yeah, basically there is a small visual bug